ui/components/ui/detailed-error-message/detailed-error-message.component.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DetailedErrorMessage = (props, context) => {
+  const { errorMessage, errorKey, onErrorMessageClick, linkText } = props;
+  const error = errorKey ? context.t(errorKey) : errorMessage;
+
+  return (
+    <div className="error-message">
+      <img
+        src="./images/alert-red.svg"
+        alt=""
+        className="error-message__icon"
+      />
+      <div className="error-message__text">
+        {error}
+        {linkText ? (
+          <a className="error-message__link" onClick={onErrorMessageClick}>
+            {` ${linkText}`}
+          </a>
+        ) : (
+          ''
+        )}
+      </div>
+    </div>
+  );
+};
+
+DetailedErrorMessage.propTypes = {
+  errorMessage: PropTypes.string,
+  errorKey: PropTypes.string,
```

Comment:
I think we should omit this prop, since we want to discourage the use of "error keys" (because they are incompatible with the strict mode of our verify-locales linter) and none are currently used with this component.